Enigmatry.Entry.AspNetCore.TestUtils/Enigmatry.Entry.AspNetCore.TestUtils.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsPackable>true</IsPackable>
+    <Authors>szi</Authors>
+    <Product>Enigmatry.Entry.AspNetCore.TestUtils</Product>
```

Comment:
This is a new project. All classes from Enigmatry.Entry.AspNetCore.Tests have been moved here. The project Enigmatry.Entry.AspNetCore.Tests is now test project for Enigmatry.Entry.AspNetCore and Enigmatry.Entry.AspNetCore.SampleApp

Reply:
Don't forget to remove the Authors field, otherwise it will overwrite the values from Directory.Build.Props